ci/travis/ci.sh

```diff
@@ -200,6 +200,9 @@ test_cpp() {
   bazel build --config=ci //cpp:all
   # shellcheck disable=SC2046
   bazel test --config=ci $(./scripts/bazel_export_options) --test_strategy=exclusive //cpp:all --build_tests_only
+  # run cluster mode test with external cluster
+  bazel test //cpp:cluster_mode_test --test_arg=--external-cluster=true --test_arg=--redis_password="1234" \
```

Comment:
What's the purpose to set both `redis_password` and `ray-redis-password`? Should we use `redis_password` or `redis-password`?

Reply:
I guess what you want to achieve is to start a local Ray cluster and then connect to it, right?  I think this way is much more clear:

```
ray start --head ...
bazel test //cpp:cluster_mode_test --test_arg=--ray-address=...
ray stop
```

Reply:
"ray-redis-password" is used by internal config, and "redis_password" is used by test code.

Reply:
If I merge this two, the test will be no sense.

Reply:
`ray start --head ...
bazel test //cpp:cluster_mode_test --test_arg=--ray-address=...
ray stop`
About this, I think make it in one process in more convenient. We will add more tests like this, such as two nodes test.

Reply:
That makes sense.